gtk/src/lib.rs

```diff
@@ -102,20 +112,20 @@
 //! - Any constructor or free function will panic if called before `init` or on
 //! a non-main thread.
 //!
-//! - Any `&str` or `&Path` parameter with an interior null (`\0`) character will
+//! - Any [&str] or [&Path](std::path::Path) parameter with an interior null (`\0`) character will
 //! cause a panic.
 //!
 //! - Some functions will panic if supplied out-of-range integer parameters. All
 //! such cases will be documented individually but they're not yet.
 //!
-//! **A panic in a closure will abort the process.**
+//! **A panic in a closure will abort the main thread.**
```

Comment:
That's not true. It will actually kill the whole process

Reply:
I think you can spawn a thread an do the init there? But I can still revert it.

Reply:
I was thinking about specifying "in a closure" a bit more. The sentence as it is seems a bit unclear.

Reply:
Any panic in a signal handler closure or any other closure passed to bindings functions will directly kill the whole process. The panic would unwind until an `extern "C"` function, and there it would then call `abort()`.

With current stable Rust it's actually UB (but in almost all cases does exactly that already) but next (or next-next?) release it's guaranteed to do that.

Reply:
But yes, making the "in a closure" part a bit more clear is a good idea :)